jerry-debugger/jerry-client-ws.html

```diff
@@ -972,15 +972,25 @@
       return true;
     }
 
-    if (args[2] == "")
+    var ipAddr = args[2];
+    var PORT = "5001";
+
+    if (ipAddr == "")
     {
-      appendLog("IP address expected");
-      return true;
+      ipAddr = "localhost";
     }
 
-    appendLog("Connect to: " + args[2]);
+    if (ipAddr.match(":")) {
```

Comment:
Can we use a regex here? Something like:

    /^.*:\d+$/

If match requires whole string match then ^ and $ is not needed.